src/prachi/Assignment_10/Calculator.java

```diff
@@ -0,0 +1,60 @@
+package prachi.Assignment_10;
+
+/*Java Assignment-10 : 3rd April 2021
+Write a class called calculator with following functionality.
+a. add
+b. subtraction
+c. division
+d. multiplication
+e. printTotalAnswer.
+
+Method a,b,c,d should have 2 parameters and return int value.
+printTotalAnswer method should have 4 parameters and doesnt return anything.
+Note : do not create any instance variable.
+
+
+Can't play...communication between 2 classes...**part2 recording video
+And Return type recordings....is anyone facing same issue??
+Bhushan Shende, Today at 14:42*/
+
+
+public class Calculator {
+
+	int a;
```

Comment:
This instance variable not require to fulfil this program output,

Refer below code, How you can get same output without using this instance